modules/consensus/accept.go

```diff
@@ -202,6 +202,6 @@ func (cs *ConsensusSet) AcceptBlock(b types.Block) error {
 		return err
 	}
 	// Broadcast the new block to all peers.
-	go cs.gateway.Broadcast("RelayBlock", b)
+	go cs.gateway.Broadcast("RelayBlock", b, nil)
```

Comment:
would like to see a check in the gateway code that throws an error or panic if a developer tries to use more than 8 characters. It's easy to forget that "RelayBlock" and "RelayBlock2" are actually the same call.